### PR TITLE
Replaced sleep() with awaitility

### PR DESCRIPTION
**Pull request by Sumukh-Phalgaonkar**
Replaced all instances of thread.sleep() with awaitility.await()

### CODE REVIEW

**Review by vaibhav-yb — COMMENTED** [review]
Just wondering, will it be feasible to shift this whole Awaitility block to the `PgHelper.java` within a function, probably something like:
```java
public void waitTillRecordsAreVerified(int expectedRecordCount, long milliSecondsToWait) throws ConditionTimeoutException {
  Awaitility.await()
    .atMost(Duration.ofMillis(milliSecondsToWait))
    .ignoreExceptionsInstanceOf(SQLException.class)
    .until(() -> pgHelper.verifyRecordCount(expectedRecordCount));
}
```

The above function can simply be called in the tests without adding repeated lines of code. We can even remove the `.atLeast()` duration as well (as in the above code block) since it doesn't make much sense in our case. What do you say @vrajat ?

File: cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java [hunk 1]
```diff
@@ -136,7 +137,12 @@ public void verifyRecordsInKafka() throws Exception {
     public void verifyRecordsInPostgresFromKafka() throws Exception {
         // Adding Thread.sleep() here because apparently Awaitility didn't seem to work as expected.
         // TODO Vaibhav: Replace the Thread.sleep() function with Awaitility
-        Thread.sleep(10000);
+        // Thread.sleep(10000);
```
Comment (vrajat) [inline cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java:140]: Can you remove the comment before committing?

**Review by vrajat — APPROVED** [review]

**Comment by vrajat** [comment]
Agree that a function will make the code cleaner. @vaibhav-yb

File: cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java (hunk 1 above)
Comment (Sumukh-Phalgaonkar) [reply cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java:140]: Yes. I'll remove them

**Review by vaibhav-yb — APPROVED** [review]

**Comment by Sumukh-Phalgaonkar** [comment]
> Just wondering, will it be feasible to shift this whole Awaitility block to the `PgHelper.java` within a function, probably something like:
> 
> ```java
> public void waitTillRecordsAreVerified(int expectedRecordCount, long milliSecondsToWait) throws ConditionTimeoutException {
>   Awaitility.await()
>     .atMost(Duration.ofMillis(milliSecondsToWait))
>     .ignoreExceptionsInstanceOf(SQLException.class)
>     .until(() -> pgHelper.verifyRecordCount(expectedRecordCount));
> }
> ```
> 
> The above function can simply be called in the tests without adding repeated lines of code. We can even remove the `.atLeast()` duration as well (as in the above code block) since it doesn't make much sense in our case. What do you say @vrajat ?

**Comment by vaibhav-yb** [comment]
Reopening the PR to merge it before closing.

**Pull request merged**